### ISSUE TITLE
[BUG] Chat bug on servers with AntiPopup and DiscordSRV installed at the same time

### ISSUE THREAD

**Issue by Jumala9163** [KaspianDev/AntiPopup#75]
### Describe the bug

When AntiPopup and DiscordSRV are installed at the same time, the same message is sent twice when a message from Minecraft is sent to Discord
Disabling AntiPopup has prevented this from happening
![image](https://github.com/KaspianDev/AntiPopup/assets/67909995/194b403f-3658-49de-b5a2-06b012606b07)

### To Reproduce

1.Install AntiPopup and DiscordSRV on the server
2. Send chats in the Miencraft server

### Expected behavior

The same statement is sent twice in Discord chat

### Operating System

Docker(Pterodactyl)

### Server Software

Patina(https://github.com/PatinaMC/Patina)

### Server Version

Not Listed

### Proxy Type

Bungeecord

### Plugin List

```bash
AntiPopup, ArmorStandEditor, AuxProtect, BCLogs, BlueMap, Chairs, ClickHarvest, CoreProtect, CoreProtectTNT, DiscordSRV, Elevatorm, EntityDetection, FastAsyncWorldEdit, GeyserSkinManager, HamsterAPI, HolographicDisplays, LightHeadPlugin, LuckPerms, LunaChat, Minepacks, MobStacker, OfflineEveryone, OpenInv, OreNotifier, PlaceholderAPI, Plan, PlugManX, Polymart, ProtocolLib, SetSpawn, SitEverywhere, SneakGrow, spark, SuperVanish, TabTPS, UltimateHomes, UltimateTimber, VeinMiner, ViaBackwards, ViaVersion, WorldEditSUI, WorldGuard
```

### Server logs

https://pastebin.com/0QjYxufT

**Comment by MrNonno** [comment]
I think the issue is some conflict with other plugins

**Comment by KaspianDev** [comment]
Hmm... I used discordsrv with antipopu personally. Can you send your config here?

**Comment by KaspianDev** [comment]
Disable clickable urls, most likely that's at fault.

**Comment by Jumala9163** [comment]
Thank you very much. I will change the settings and see what happens. I will report back if there is any progress.

**Comment by Jumala9163** [comment]
It has been a while since then.
As a result, I had version 7.1 installed on my server, so I updated to 7.2 to try it out, and for some reason it seems to work fine with [clickable-urls: true] in 7.2.

**Comment by K4sum1** [comment]
I have this issue with EssentialsX Discord. If you were able to fix it with DiscordSRV in 7.2, could you fix it with EssentialsX Discord too?

**Comment by KaspianDev** [comment]
> I have this issue with EssentialsX Discord. If you were able to fix it with DiscordSRV in 7.2, could you fix it with EssentialsX Discord too?

I recommend disabling `clickable-urls` if it's on. It's a very tricky system so it causes incompabilities with such plugins, a chat plugin with the feature will do a better job.

**Comment by K4sum1** [comment]
> I recommend disabling `clickable-urls` if it's on. It's a very tricky system so it causes incompabilities with such plugins, a chat plugin with the feature will do a better job.

Yeah I have it off, just would prefer to have it on if possible. I prefer EssentialsX Discord over DiscordSRV.

**Comment by KaspianDev** [comment]
message me on discord, i'll see what i can do
